src/categorical_algebra/FinSets.jl

```diff
@@ -325,6 +325,7 @@ symbolic attributes. (Likewise, AttrVars will have to wrap Any rather than Int)
     return new(FinDomFunction(Vector{Union{AttrVar,T}}(f)), cod)
   end 
 end
+VarFunction(f::UnitRange,cod::Int) = VarFunction(FinFunction(f,cod))
```

Comment:
This function is not covered and since

```julia
UnitRange{Int} <: AbstractVector{Int}
```

I figure it is not actually needed?